Updated for 2018 date and registration form

dcgaines opened the pull request.

dcgaines committed:
Updated for 2018 date and registration form

raveious commented on _events/pi-mile.md:
```diff
@@ -38,6 +38,7 @@ Our partnerships make this event a success. The sisters of [Alpha Gamma Delta](h
 
-Courtesy of the [Inter-Residence Housing Council (IRHC)](https://www.involvement.mtu.edu/organization/IRHC), the first **100** Michigan Tech Residence Hall students who signup will receive a **half-price** discount on the registration fee. Signup before the run and receive a T-shirt at no additional cost, while supplies last.
+<!--Courtesy of the [Inter-Residence Housing Council (IRHC)](https://www.involvement.mtu.edu/organization/IRHC), the first **100** Michigan Tech Residence Hall students who signup will receive a **half-price** discount on the registration fee. Signup before the run and receive a T-shirt at no additional cost, while supplies last.-->
+The 2018 Pi Mile is scheduled for Sunday, September 23 and will begin at 10:00 AM. If you have additional questions contact dcgaines@mtu.edu.
```
Perhaps we could use `[triangle-svc@mtu.edu](mailto:triangle-svc@mtu.edu)` instead of your personal MTU email?

raveious replied:
Or at the very least, make it a hyperlink to your personal email using Markdown.

dcgaines committed:
Updated contact email 

Updated the contact email to the Service and Philanthropy email instead of my personal email.
